test_aodndata/moorings/getFeature.json

```diff
@@ -6,7 +6,7 @@
             "id": "moorings_all_map.fid--44e8da32_16d0014d48c_7b17",
             "geometry": null,
             "properties": {
-                "url":"IMOS/ANMN/NRS/NRSROT/Temperature/IMOS_ANMN-NRS_TZ_20180406T080000Z_NRSROT_FV01_NRSROT-1804-SBE39-27_END-20180817T023000Z_C-20180820T010304Z.nc",
+                "url":"IMOS/ANMN/NRS/NRSROT/Temperature/IMOS_ANMN-NRS_TZ_20180406T080000Z_NRSROT_FV01_NRSROT-1804-SBE39-27_END-20180413T064000Z_C-20200313T000000Z.nc",
```

Comment:
This things lead to a very high maintenance burden - just mock that up somehow.

Reply:
Same as for all others below

Reply:
This _is_ the mocked response from the WFS server. There's nothing to maintain. The files themselves are included in the repo.